woocommerce/woocommerce
# add uruguay states

### All Submissions:






### Changes proposed in this Pull Request:
Add States for Uruguay and proper label to field


Closes #31164.

### How to test the changes in this Pull Request:

1. add a product on your shop page
2. go to the cart -> click on next to get to the checkout
3. on the checkout choose country
4. check content of state field

### Other information:




### Changelog entry
add uruguay states

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
